news/supporting-the-future-of-rust-spotlight-on-teaching-initiatives.md

```diff
@@ -0,0 +1,66 @@
+
+The project's modular structure lets instructors create custom learning tracks for their specific needs. As an open source permissively licensed project, teach-rs enables educators to share and improve resources, making introducing Rust instruction into their programs more accessible.
+
+Many institutions now use teach-rs in their courses, including the Slovak University of Technology, [<u>RustIEC</u>](https://en.rustiec.be/workshops.html) (a collaboration between Vrije Universiteit Brussel and KU Leuven), and the [<u>University Politehnica of Bucharest</u>](https://upb-cs-rust.github.io/101-rs/). At the time of this writing, teach-rs has nearly 3000 stars on [<u>GitHub</u>](https://github.com/trifectatechfoundation/teach-rs/stargazers).
+
+Teach-rs is a [Trifecta Tech Foundation](https://trifectatech.org/initiatives/education/) initiative maintained by engineers, Henk Oordt, Tamme Dittrich, and Marc Schoolderman. The project has received support from [Rust-Edu](https://rust-edu.org/) in the past and continues to collaborate with Rust-Edu to improve and expand the course materials.
+
+Those interested in learning more or contributing to teach-rs can visit the [<u>teach-rs matrix channel</u>](https://matrix.to/#/#teach-rs:matrix.org) or the teach-rs [<u>GitHub repository</u>](https://github.com/trifectatechfoundation/teach-rs).
+
+### A Guide To Implementing Rust In Your Curriculum
+
+Under the guidance of The Rust Foundation's Global Rust Coordinator and [<u>Rust Nation UK's</u>](https://www.rustnationuk.com/) organizer Ernest Kissiedu, [<u>Mordecai Etukudo</u>](https://github.com/martcpp) (Mart) has developed a [<u>guide</u>](https://www.notion.so/Step-By-Step-Guide-To-Help-You-Implement-Rust-Effectively-In-Your-University-ab5cfd8df94a4ae0ac9d8bea9d38d2fe?pvs=4) to help educational institutions adopt Rust in their systems. This resource walks organizations through the entire implementation process, from initial assessment to community engagement.
+
+The guide addresses key phases of Rust adoption, including curriculum assessment, course content design, teaching resource development, faculty training, and community integration.
+
+Designed to complement existing resources like Rust-Edu and teach-rs, it is a living document that welcomes [<u>contributions and improvements</u>](https://www.notion.so/Step-By-Step-Guide-To-Help-You-Implement-Rust-Effectively-In-Your-University-ab5cfd8df94a4ae0ac9d8bea9d38d2fe?pvs=4) from the broader Rust community.
+
+## Growing the Rust Talent Pool Together
+
+The Rust Foundation's mission includes ensuring a healthy Rust ecosystem, which requires a growing pool of well-trained developers. The initiatives highlighted here show how our community is working to meet this need, creating new pathways for learning Rust.
```

Comment:
~~The Rust Foundation's mission includes~~ **The Rust Foundation is dedicated to** ensuring a healthy Rust ecosystem, which ~~requires~~ **depends** on a growing pool of well-trained developers **to thrive**.